packages/cli/lib/cmds/downloadFramework.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const downloadFrameworkProvider = require('@ampproject/toolbox-download-framework');
+
+async function downloadFramework(args, _) {
+  const {ampUrlPrefix, clear, dest, rtv} = args;
```

Comment:
We need to check if dest has been provided. Would it make sense to use `rtv/$RTV_VERSION/` as the default value for `dest`? This way it'd be easy to replicate the cache fs structure.